api/src/main/java/org/openmrs/module/providermanagement/ProviderPersonRelationship.java

```diff
@@ -0,0 +1,29 @@
+package org.openmrs.module.providermanagement;
+
+import org.openmrs.Person;
+import org.openmrs.Relationship;
+import org.openmrs.RelationshipType;
+
+public class ProviderPersonRelationship {
+
+    Person person = null;
+    String identifier = null;
+    Integer objectId = null;
+    Relationship relationship= null;
+    RelationshipType relationshipType = null;
+
+    public ProviderPersonRelationship() {}
+
+    public ProviderPersonRelationship(Person person,
```

Comment:
Let's documentation that this is basically a utility domain object for use by the coreapps provider management functionality (if I'm correct about that?)  It also might sense to put this in a sub-package and not the top-level provider management package, since it is of lesser importance, so someone coming into the module for the first time won't be distracted/confused by it--does that make sense?  Not sure exactly what the package should be called though.